### DISCUSSION TITLE
EF Core Blazor

### DISCUSSION THREAD

**Discussion by fredcdf** [balta-io/forum#230] [Tira Dúvidas] [answered]
Executo uma leitura de dados de um SQL server local na minha máquina através de um projeto de console e imprime na tela os dados de uma tabela simples. 

Quando tento ler os mesmos dados usando o mesmo código mas apenas mudando o console para um projeto blazor , dá erro na execução quando no ponto exato da leitura da variável da injeção de dependência. 
Como resolvo essa conexão ? Será um problema de autenticação exigida pelo navegador para acessar meu SQL ? 

Já desliguei meu firewall, mudei de navegador, e o erro continua. Uso .net 7.0 atualizado! Visual Studio 2022 atualizado! Não consigo fazer EF Core funcionar com nenhum projeto Blazor. Sempre dá um erro de execução no ato de carregar a variável injetada!

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @fredcdf qual erro dá? Tem um print ou detalhes do erro?

**Reply by fredcdf** [reply]
Tentei em vários navegadores, https, http, IIS no modo de execução... e nada resolve. Fazendo a impressão de dados num projeto de console , funciona como esperado, porém quando uso pelo navegador dá esse erro! Variei vários tipos de ProjetoBlazor, Wasm, Servidor, sempre dá esse erro! 

<img width="483" alt="image" src="https://user-images.githubusercontent.com/497155/213083140-4205c376-4289-48ec-84af-2dc554f49314.png">


<img width="358" alt="image" src="https://user-images.githubusercontent.com/497155/213083379-9ace7ea2-0b0e-46d6-9420-b39155055108.png">

<img width="291" alt="image" src="https://user-images.githubusercontent.com/497155/213083564-44f0d019-b67a-4090-b862-8e6ace53690b.png">

<img width="1093" alt="image" src="https://user-images.githubusercontent.com/497155/213083617-53fa0ca1-9d25-4493-a004-3b3ac37c630f.png">

Eu gerei a Lib apra conexão com SQL Server usando o comando:
Scaffold-DbContext  "Data Source=DESKTOP; Initial Catalog=EPE_BASE_Dev; Integrated Security=True; User ID=sa;Password=fred" Microsoft.EntityFrameworkCore.SqlServer -OutputDir Modelos  -Schema 'dbo' -DataAnnotations

Tentei muita coisa e nada faz funcionar. Estou fazendo teu curso mas já queria fazer a conexão do SQL Server com Blazor , fiz algo muito simples e mesmo assim não consegui fazer funcionar. Meu VS studio 2022 está atualizado e com .net 7 também atualizado rodando com tudo em net. 7.0. Windows 11 Pro. 


<img width="537" alt="image" src="https://user-images.githubusercontent.com/497155/213085220-5c85cb81-8b29-445b-8068-5d119f2826af.png">

**Reply by andrebaltieri** [reply]
@fredcdf não consegui ver o erro que está tendo... Tem alguma mensagem, algum erro específico? Debugando aparece algo?
